README.rst

```diff
@@ -78,3 +78,24 @@ This will generate a new migration script. Then run:
 To apply the migration.
 
 For a full migration command reference, run ``python manage.py db --help``.
+
+
+Docs
+----
+
+To build the docs:
+
+move to the `slate` directory:
+::
+
+    cd slate/
+
+then build the docs with middleman:
+::
+
+    bundle exec middleman build
```

Comment:
OK following these instructions, in `skate/`, building the docs gives errors when building.

```
$ bundle exec middleman build
   identical  /Users/mjperrone/Development/GameCenter/gamecenter/static/docs/stylesheets/print.css
   identical  /Users/mjperrone/Development/GameCenter/gamecenter/static/docs/stylesheets/screen.css
   identical  ../gamecenter/static/docs/images/navbar.png
   identical  ../gamecenter/static/docs/images/logo.png
   identical  /Users/mjperrone/Development/GameCenter/gamecenter/static/docs/fonts/slate.svg
   identical  ../gamecenter/static/docs/fonts/slate.woff
   identical  ../gamecenter/static/docs/fonts/slate.woff2
   identical  ../gamecenter/static/docs/fonts/slate.ttf
   identical  ../gamecenter/static/docs/fonts/slate.eot
       error  ../gamecenter/static/docs/javascripts/all_nosearch.js
       error  ../gamecenter/static/docs/javascripts/all.js
      update  /Users/mjperrone/Development/GameCenter/gamecenter/static/docs/index.html
There were errors during this build, re-run with `middleman build --verbose` to see the full exception.
```

The javascript files aren't made/updated so the ToC and the tabbed code view don't work in the generated docs. Did you do anything different for setup than what you describe in this readme?